### PR TITLE
PI-12091: removed logs for health check requests

### PR DESCRIPTION
**Pull request by tedturner29**
[JIRA ticket](https://appdirect.jira.com/browse/PI-12091)

Removed logs for health check requests to reduce the number of unnecessary logs in production.

### CODE REVIEW

File: src/main/resources/logback-access.xml [hunk 1]
```diff
@@ -13,6 +13,13 @@
 
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
+            <evaluator>
+                <expression>event.getRequestURL().contains("/health")</expression>
```
Comment (amertkara) [inline src/main/resources/logback-access.xml]: should you add a `return` statement as mentioned here https://logback.qos.ch/manual/filters.html#JaninoEventEvaluator?

Comment (tedturner29) [reply src/main/resources/logback-access.xml]: I believe that's only necessary for multi line expressions

File: pom.xml [hunk 2]
```diff
@@ -58,6 +58,11 @@
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-access</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.codehaus.janino</groupId>
+			<artifactId>janino</artifactId>
+			<version>3.0.7</version>
```
Comment (amertkara) [inline pom.xml]: minor: seems like there is a recent 3.0.8 version

**Comment by hooriak** [comment]
LGTM.

**Pull request merged**